### DISCUSSION TITLE
Status codes for ConnectionFailed

### DISCUSSION THREAD

**Discussion by DanCSF** [lostisland/faraday#1430] [Help]
* Faraday Version: 0.17.4
* Ruby Version: 2.7.4

Can anyone please tell me what are the exact status codes for ConnectionFailed exception?

Thank You

**Comment by iMacTia** [comment]
@DanCSF do you mean HTTP status codes (e.g. 200, 400, ...)?
In that case, there won't be any in case of `ConnectionFailed`. That's because those statuses are generated by the server, and `ConnectionFailed` means a connection with the server could not be established for some reason.

(NOTE: I've converted your issue into a Discussion since it seemed more like a question)
